nncf/experimental/post_training/algorithms/min_max_quantization.py

```diff
@@ -0,0 +1,108 @@
+"""
```

Comment:
Why can we place this file into the quantization subfolder? It looks like a general abstract algorithm for the backend-relative ones, isn't it?

Reply:
Yes, placed under quantization folder